ntbs-service/Pages/Notifications/Edit/SocialContextAddresses.cshtml.cs

```diff
@@ -35,7 +35,7 @@ protected override async Task ValidateAndSave()
         
         protected override IActionResult RedirectToCreate()
         {
-            return RedirectToPage("./Items/NewSocialContextAddress", new { NotificationId });
+            return RedirectToPage("./Items/NewSocialContextAddress", "", new { NotificationId }, "social-context-address-form");

```

Comment:
Nancy wanted this anchor so we immediately jump to the input fields, as on small screens you have to do a lot of scrolling otherwise